Upload only 3lap 1

salvoch opened the pull request:
Changed mk64.py to upload only the fastest 3-lap record instead of any top 5 record.

salvoch committed:
Updated config yaml to have the -

salvoch committed:
Merge branch 'main' of github.com:SpeedRunTooked/mk64-watcher into main

salvoch committed:
Changed loop to only send mk64.py
Changed the loop in mk64.py in compare_records
Instead of iterating through all 5 records, we only
look at the record at [0] (the top record).

salvoch committed:
Removed config.yml
did a git rm config.yml to untrack the config,
this should not happen in the future since it's on the gitignore

gabranches commented:
LGTM 👍

jmct commented on src/utils/mk64.py:
```diff
@@ -5,20 +5,15 @@
+
+        three_lap_record = new_times['track-record'].records[0]
+        if old_times['track-record'].records[0] != three_lap_record and three_lap_record != None:
+            # Commented these out because not being used atm
```
Why'd you keep these comments (that represent code we don't need), but delete the explanatory comments?

jmct reviewed (changes requested):
Code is simpler, which is always nice!

Just add back the explanation of why we only check whether `nr_re` is `None`. It might be obvious to us now, but it took a bit of thinking on the day and future us won't remember.

salvoch committed:
Added comments explaining the new record, removed old comment

Added the comments explaining why we skip if new record = None
Also deleted commented out code referencing code that is no longer needed

jmct reviewed (approved):
Awesome, thanks!
